### DISCUSSION TITLE
Feature request on tasks

### DISCUSSION THREAD

**Discussion by shiva-sandupatla** [community/community#4843] [Projects and Issues]
Hi, 

It is better, when I created tasks in issues and if other member in my organization checked any task or unchecked task history can be found under issue history. Hope you got my point.

Regards, shiva

**Comment by belauzas** [comment]
- [x] Do you mean
- [ ] these tasks?

If so, I believe, it would be really helpful to have such check/uncheck history with who and when did it 👍

**Reply by shiva-sandupatla** [reply]
Yes these tasks. As you said it will be really helpful

**Comment by nexushoratio** [comment]
I believe this information can be found in the edit history of the initial comment (not the issue history).  From playing around just now, it looks like the whole task list gets marked as changed in the UI, but you have to visually figure out what changed.  But it looks like who/when is available.  On the other hand, turning them into child issues probably gives more control.
